src/gluonnlp/data/classification.py

```diff
@@ -347,3 +616,18 @@ def get_dataset(self, segment='train'):
             Dataset segments. Options are 'dev', 'test', 'train'
         """
         return BaiduErnieChnSentiCorp(segment)
+
+tasks = {
+    'MRPC': MRPCTask(),
+    'QQP': QQPTask(),
+    'QNLI': QNLITask(),
+    'RTE': RTETask(),
+    'STS-B': STSBTask(),
+    'CoLA': CoLATask(),
+    'MNLI': MNLITask(),
+    'WNLI': WNLITask(),
+    'SST': SSTTask(),
+    'XNLI': XNLITask(),
+    'LCQMC': LCQMCTask(),
+    'ChnSentiCorp': ChnSentiCorpTask()
+}
```

Comment:
Instead of exposing `tasks` directly, what about not show it in API but rather have a get_task() API that takes the task name as the input and return the corresponding task. We don't want user to access `tasks` directly and modify it. It will `return tasks[task.lower()].copy()`